App-Service-Environment-AzFirewall/nestedtemplates/vnet.gov.json

```diff
@@ -0,0 +1,666 @@
+                        "properties": {
+                            "addressPrefix": "13.72.180.105/32",
+                            "nextHopType": "Internet"
+                        }
+                    },
+                    {
+                        "Name": "23.97.0.17-route",
+                        "properties": {
+                            "addressPrefix": "23.97.0.17/32",
+                            "nextHopType": "Internet"
+                        }
+                    },
+                    {
+                        "Name": "23.97.16.184-route",
+                        "properties": {
+                            "addressPrefix": "23.97.16.184/32",
+                            "nextHopType": "Internet"
+                        }
+                    },
+                    {
+                        "name": "DefaultRoute",
+                        "properties": {
+                            "addressPrefix": "0.0.0.0/0",
+                            "nextHopType": "VirtualAppliance",
+                            "nextHopIpAddress": "[variables('azureFireWallPrivateIp')]"
+                        }
+                    }
+                ]
+            }
+        },
+        {
+            "apiVersion": "2015-06-15",
+            "type": "Microsoft.Network/routeTables",
+            "name": "[parameters('azureFirewallRouteTableName')]",
+            "location": "[parameters('location')]",
+            "tags": "[parameters('tags')]",
+            "properties": {
+                "routes": [
+                    {
+                        "name": "default-route",
+                        "properties": {
+                            "addressPrefix": "0.0.0.0/0",
+                            "nextHopType": "Internet"
+                        }
+                    }
+                ]
+            }
+        },
+        {
+            "apiVersion": "2017-06-01",
+            "type": "Microsoft.Network/virtualNetworks",
+            "name": "[parameters('vnetResourceName')]",
+            "location": "[parameters('location')]",
+            "tags": "[parameters('tags')]",
+            "properties": {
+                "addressSpace": {
+                    "addressPrefixes": [
+                        "[parameters('vnetAddressRange')]"
+                    ]
+                },
+                "subnets": [
+                    {
+                        "name": "[parameters('aseSubnetName')]",
+                        "properties": {
+                            "addressPrefix": "[parameters('aseSubnetAddressRange')]",
+                            "networkSecurityGroup": {
+                                "id": "[variables('nsgId')]"
+                            },
+                            "routeTable": {
+                                "id": "[resourceId('Microsoft.Network/routeTables', parameters('aseRouteTableName'))]"
+                            },
+                            "serviceEndpoints": [
```

Comment:
these should be a parameter since there's nothing in this workload that is using them